charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/configmap-observability.yaml

```diff
@@ -1,55 +0,0 @@
-{{- if not .Values.gep19Monitoring }}
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cloud-controller-manager-observability-config
```

Comment:
Not introduced with this PR but we could delete https://github.com/gardener/gardener-extension-provider-openstack/blob/46af0bca70a68266fb6cc26632f399aa8377d380/pkg/controller/controlplane/valuesprovider.go#L138 and https://github.com/gardener/gardener-extension-provider-openstack/blob/46af0bca70a68266fb6cc26632f399aa8377d380/pkg/controller/controlplane/valuesprovider.go#L348-L351.
Introduced with https://github.com/gardener/gardener-extension-provider-openstack/pull/766.